README.md

```diff
@@ -143,6 +158,45 @@ $ sirun baz-test.json >> results.ndjson
 $ cat results.ndjson | sirun --summarize > summary.json
 ```
 
+Each line of output in one of these `.ndjson` files is a complete JSON document.
+Here's an example of one of these lines of output, though whitespace has been added for readability:
+
+```json
+{
+    "name": "foobar",
+    "variant": "with-baz",
+    "iterations": [
+        {
+            "cpu.pct.wall.time": 18.138587328535383,
+            "max.res.size": 66956,
+            "system.time": 766091,
+            "user.time": 1552557,
+            "wall.time": 12782958
+        },
+        {
+            "cpu.pct.wall.time": 18.029851850045276,
+            "max.res.size": 66480,
+            "system.time": 720491,
+            "user.time": 1571242,
+            "wall.time": 12710770
+        }
+    ]
+}
+```
+
+- **`name`**: This is the same `name` value from the configuration file.
+- **`variant`**: This is the object key from the configuration file's `variations` list.
+- **`iterations`**: These are the statsd metrics from different runs and contain raw data
+  - **`max.res.size`**: Bytes (B) maximum Resident Set Size (RSS), aka the highest RAM usage
+  - **`system.time`**: Microsecond (μs) amount of time spent in kernel code
+  - **`user.time`**: Microsecond (μs) amount of time spent in application code
+  - **`wall.time`**: Microsecond (μs) amount of time the overall iteration took
+  - **`cpu.pct.wall.time`**: Percentage (%) of time where the program was waiting (not system or user time)
```

Comment:
It's actually the opposite: percentage of time where the program was _not_ waiting.